doc/api/process.md

```diff
@@ -1187,6 +1187,38 @@ setTimeout(() => {
 }, 1000);
 ```
 
+## process.hrtime.bigint([time])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `time` {bigint} The result of a previous call to `process.hrtime.bigint()`
+* Returns: {bigint}
+
+The `bigint` version of the [`process.hrtime()`][] method. If `time` is not
+specified, it returns the current high-resolution real time in a `bigint`.
+
+If `time` is specified, it must be the result of a previous
+`process.hrtime.bigint()` call. The returned result will be the difference
+between the current time and the specified time.
```

Comment:
While useful for the array-based method, I don’t think this feature is that useful now one can just subtract  the times directly…

Reply:
@TimothyGu Yeah come to think of it, it does seem unnecessary. I'll remove it.